app/src/test/java/com/example/samplegallery/Utilities/LruBitmapCacheTest.java

```diff
@@ -0,0 +1,41 @@
+package com.example.samplegallery.Utilities;
+
+import android.content.Context;
+import android.icu.util.RangeValueIterator;
+import android.util.DisplayMetrics;
+
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.*;
+
+/**
+ * Created by parth on 1/22/2017.
+ */
+
+public class LruBitmapCacheTest {
+
+    @Mock
+    Context mMockContext;
+
+    @Test
+    public void getCacheSize_isCorrect() throws Exception {
+
+        mMockContext = Mockito.mock(Context.class, Mockito.RETURNS_DEEP_STUBS);
+
+        DisplayMetrics dm = Mockito.mock(DisplayMetrics.class, Mockito.RETURNS_DEEP_STUBS);
+
+        dm.widthPixels = 50;
+        dm.heightPixels = 100;
+
+        Mockito.when(mMockContext.getResources().getDisplayMetrics()).thenReturn(dm);
+
```

Comment:
Add spaces between the operands in your 'equalTo' clause, e.g (dm.widthPixels * dm.heightPixels * 4 * 3)